### PR TITLE
[AA-1734] Subscription usage chart

### PR DESCRIPTION
**Pull request by skochay330**
New subscription usage page

Jira issue: https://issues.redhat.com/browse/AA-1734 

Screenshot 2023-08-24 at 5 09 44 PM

### CODE REVIEW

File: framework/PageDashboard/PageDashboardChart.tsx [hunk 1]
```diff
@@ -83,9 +85,20 @@ export function PageDashboardChart(props: {
...
+                datum: {
+                  x: string | number;
+                  y: string | number;
+                  hosts_added?: string | number;
+                  hosts_deleted?: string | number;
+                };
+              }) => {
                 const datum = point.datum;
-                return `${datum.x}: ${datum.y}`;
+                return yLabel == 'Unique hosts' && datum.hosts_added !== undefined
```
Comment (ZitaNemeckova) [inline framework/PageDashboard/PageDashboardChart.tsx:97]: I thin this should be translated.

Comment (jamestalton) [reply framework/PageDashboard/PageDashboardChart.tsx:97]: These chart usage specific strings should not be in the generic PageDashboardChart.

File: framework/PageDashboard/PageDashboardChart.tsx [hunk 2]
```diff
@@ -83,9 +85,20 @@ export function PageDashboardChart(props: {
           }
           containerComponent={
             <ChartVoronoiContainer
-              labels={(point: { datum: { x: string | number; y: string | number } }) => {
+              labels={(point: {
+                datum: {
+                  x: string | number;
+                  y: string | number;
+                  hosts_added?: string | number;
```
Comment (jamestalton) [inline framework/PageDashboard/PageDashboardChart.tsx:92]: These chart usage specific value should not be in the generic PageDashboardChart.

Maybe make this a tooltip details?
```
tooltipDetails: string[] 
```

Then we could render those under the item in the tooltip.

Note: The tooltips were enhanced in this [PR](https://github.com/ansible/ansible-ui/pull/811) to use the group label in the tooltip.

File: frontend/awx/analytics/subscription-usage/SubscriptionUsage.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        <PageDashboardCard
+          title={t('Subscription Compliance')}
+          width="xxl"
+          height="xl"
+          headerControls={
+            <Flex spaceItems={{ default: 'spaceItemsNone' }} style={{ gap: 8 }}>
+              <FlexItem>
+                <Text component="small">
+                  Last recalculation date:{' '}
+                  {systemData.data?.HOST_METRIC_SUMMARY_TASK_LAST_TS.slice(0, 10)}
```
Comment (jamestalton) [inline frontend/awx/analytics/subscription-usage/SubscriptionUsage.tsx:73]: Ran into an error here
> Uncaught error: TypeError: Cannot read properties of undefined (reading 'slice')

**Review by jamestalton — APPROVED** [review]

**Pull request merged**